services/nginx/conf.d/shift.conf

```diff
@@ -70,6 +70,16 @@ server {
         alias /opt/legacy/cal;
     }
 
+    # handle images of the standard shift format 
+    # ex. /eventimages/9248-124.png
+    #
+    location ~ /eventimages/(\d+)(?:-\d+).(\w+) {
```

Comment:
Is an image guaranteed to have an extension? In most cases it will, but based on a quick test it looks like I can still upload a valid JPG (for example) without an extension. There may be (probably are?) existing bugs related to this, but wondering regex here that assumes an extension could be a new one. Probably an acceptable case for now either way.

Reply:
oh wow, interesting! 

so, the existing code ( in [Event.php](
https://github.com/shift-org/shift-docs/blob/b25c39d574bd5fe852185cae750713dbe7b9b57c/backend/models/Event.php#L253) )  always tries to add a "." 

```
// What the name should be
        $t = pathinfo($old_name);
        $ext = $t['extension'];
        $new_name = "$id.$ext";
```

so i'm guessing that the url of those images would be something like ` /eventimages/9248.` and the cache busting version (`/eventimages/9248-124.` ) would still work but would be a good thing to follow up on and verify.